avsomr/msmrwriter.cpp

```diff
@@ -0,0 +1,72 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "msmrwriter.h"
+
+#include <QBuffer>
+
+#include "libmscore/score.h"
+
+#include "avslog.h"
+#include "msmrfile.h"
+
+using namespace Ms::Avs;
+
+MsmrWriter::MsmrWriter()
+      {
+
+      }
```

Comment:
Why not a one-liner here?

Reply:
Is this important or can we leave it up to the author?

Reply:
not important maybe, but different from anywhere else in the code, and at least one person (me) stubled accross it while reading the code

Reply:
ok, removed. Although I prefer not to do one-liners - it’s easier to debug, to insert code for debugging, it’s easier to modify, it’s easier to see the changes (if there is a change in the one-liner, then in the git blame the whole constructor will be the authorship of another developer, as he added, but did not change :) )